cli/src/semgrep/commands/ci.py

```diff
@@ -287,6 +302,7 @@ def ci(
             console.print(Title("Connection", order=2))
             metadata_dict = metadata.to_dict()
             metadata_dict["is_sca_scan"] = supply_chain
+            metadata_dict["is_secrets_scan"] = beta_testing_secrets
```

Comment:
Is there a schema somewhere for this metadata?

Reply:
No schema for this.

Reply:
We might want to define it at some point in semgrep-interfaces/ is this dict becomes heavily (ab)used.